test/assets/quality_gates_standalone_slo_step1.yaml

```diff
@@ -21,8 +21,8 @@ objectives:
   - sli: "throughput"
     pass:
       - criteria:
-          - "<=+20%"
-          - ">=-20%"
+          - "<=+100%"
```

Comment:
Why have these criteria been increased?

Reply:
they were too restrictive - sometimes the tests were failing simple because the throughput of a test run was a little bit (i.e. more than 20% higher) than in a previous test run - in my opinion, this should not lead to a failure because the goal of the test is only to verify that the mechanics of retrieving SLIs and evaluating them are working